LibraryWebApp/Views/Author/Index.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@model IEnumerable<LibraryWebApp.Models.Author>
+
+<div class="container p-3">
+    <div class="row pt-4">
+        <div class="col-6">
+            <h2 class="text-primary">Authors List</h2>
+        </div>
+        <div class="col-6 text-right">
+            <a asp-controller="Author" asp-action="Create" class="btn btn-primary">Add new Author</a>
+        </div>
+    </div>
+
+
+    <br /><br />
+
+    @if (Model.Count() > 0)
+    {
+        <table class="table table-bordered table-striped" style="width:55%" align="center">
+            <thead>
+                <tr>
+                    <th>
+                        <label asp-for="@Model.ToList()[0].Id"></label>
```

Comment:
This doesn't look good. Whe using this aproach you have to convert whole collection 3 times in this table. This would be very costly if your collection had a lot of elements